.github/workflows/publish-rust.yaml

```diff
@@ -0,0 +1,64 @@
+name: publish
+
+on:
+  workflow_call:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  rust-publish:
+    runs-on: ubuntu-latest
+    container:
+      image: rapidsai/ci-conda:latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check if release build
+        id: check_release
+        run: |
+          if rapids-is-release-build; then
+            echo "is_release=true" >> $GITHUB_OUTPUT
+          else
+            echo "is_release=false" >> $GITHUB_OUTPUT
+          fi
+      - name: Install Rust (cargo)
+        if: steps.check_release.outputs.is_release == 'true'
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
Do we need to install rust here ? IIRC the rust build install's rust/cargo through conda w/ https://github.com/rapidsai/cuvs/blob/af3a4ace680fd8fb0fb20b765926411254cea604/dependencies.yaml#L91 - and this should be happening on line 41 below

Reply:
Fixed!